### PR TITLE
[TC-813] add sections and roles component

### PR DESCRIPTION
**Pull request by fw-noel**
[TC-813](https://bcdevex.atlassian.net/browse/TC-813)

Objective:
Frontend for displaying section and roles
Component should be able to handle multiple types (eg. functions is now through the same component, skills/languages/certs might be doable as well)
Integration to come later

[TC-813]: https://bcdevex.atlassian.net/browse/TC-813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/src/components/profile/sections-roles/MemberSectionAndRoles.tsx [hunk 1]
```diff
@@ -1,64 +1,51 @@
...
+  columns: { key: string; name: string }[];
+  data: {
+    id: number;
+    [key: string]: string | number;
+  }[];
+  removeRow: (id: number) => void;
+}) => {
+  const firstChoiceSection = preferences?.first;
+  const secondChoiceSection = preferences?.second;
+  // const thirdChoiceSection = preferences?.third;
```
Comment (chelsea-EYDS) [inline frontend/src/components/profile/sections-roles/MemberSectionAndRoles.tsx:31]: Are we adding a third option?

**Review by chelsea-EYDS — APPROVED** [review]
Looks good - just a couple of commented out props but  if they're needed all good

File: frontend/src/components/profile/sections-roles/MemberSectionAndRoles.tsx (hunk 1 above)
Comment (fw-noel) [reply frontend/src/components/profile/sections-roles/MemberSectionAndRoles.tsx:31]: Yes

File: frontend/src/components/tabs/Availability.tsx [hunk 2]
```diff
@@ -2,47 +2,69 @@ import { Tabs, TabsHeader, TabsBody, Tab, TabPanel } from '@material-tailwind/re
...
   MemberSectionsAndRoles,
-  MemberProfileFunctions,
   MemberSkillsAndCertifications,
 } from '@/components';
 import {
   Program,
-  type BcwsRoleInterface,
-  type FunctionType,
+  // type BcwsRoleInterface,
+  // type FunctionType,
```
[the remark this answers is not in the document]
Comment (fw-noel) [reply frontend/src/components/tabs/Availability.tsx:10]: When I add the integration for editing these fields, yes I believe so

**Pull request merged**